chapter-2.md

```diff
@@ -741,6 +741,37 @@ test "precision" {
 }
 ```
 
+# Advanced Function Invocation
+Zig offers a `@call` builtin function which gives you more flexibility with which how a function is
+invoked. There are options to enable or disable tail call optimizations, enable or disable inlining,
+as well as force compile time generation.
+
+Example:
+
+```zig
+const assert = @import("std").debug.assert;
+
+// These functions are effectively the same but are defined and invoked differently.
```

Comment:
Thats vague. Why are they the same and how invoked differently? Explaining them sounds more understandable to me. Then likely the comments below can also reduced to the key differences.